common/test/acceptance/pages/lms/teams.py

```diff
@@ -229,10 +240,15 @@ def sort_teams_by(self, sort_order):
         ).click()
         self.wait_for_ajax()
 
+    def click_all_topics_link(self):
```

Comment:
A "click all topics" method also exists in TeamsPage (which I would expect all the tests to have access to). I did not use the wait_for_ajax though, as I did not find it to be necessary.